transforms/dialog-full-screen-app-wrapper/dialog-full-screen-app-wrapper.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Add <AppWrapper> around the children on <DialogFullScreen>
+ * <DialogFullScreen><AppWrapper>{ children }</ AppWrapper></ DialogFullScreen>
+ */
+import registerMethods from "../registerMethods";
+
+function transformer(fileInfo, api, options) {
+  const j = api.jscodeshift;
+  registerMethods(j);
+
+  const addImport = () => {
+    // DialogFullScreen import
+    const dialogFullScreenImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/dialog-full-screen"
+      }
+    });
+
+    const appWrapperImportString = "carbon-react/lib/components/app-wrapper";
+
+    // AppWrapper import
+    let appWrapperImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: appWrapperImportString,
+      },
+    });
+
+    // if no AppWrapper import exists already add one
+    if (!appWrapperImport.size()) {
+      appWrapperImport = j.importDeclaration(
+        [j.importDefaultSpecifier(j.identifier("AppWrapper"))],
+        j.stringLiteral(appWrapperImportString)
+      );
+    
+      // insert AppWrapper import after DialogFullScreen import
+      dialogFullScreenImport.insertAfter(appWrapperImport);
+
+      return true;
+    } else {
+      return false;
+    }
+  };
+
+  const addAppWrapper = (dialogFullScreen, addedImport) => {
+    let appWrapperIdentifier = "AppWrapper";
+
+    // if there already was an import
+    if (!addedImport) {
+      const appWrapperImport = root.find(j.ImportDeclaration, {
+        source: {
+          type: "Literal",
+          value: "carbon-react/lib/components/app-wrapper",
+        },
+      });
+
+      // find identifier used for input
+      appWrapperIdentifier = appWrapperImport.paths()[0].value.specifiers[0].local.name;
+    }
+    
+     // AppWrapper opening tag
+    const appWrapper = j(dialogFullScreen).find(j.JSXOpeningElement, {
+      name: {
+        type: "JSXIdentifier",
+        name: appWrapperIdentifier,
+      },
+    });
+
+    // if no AppWrapper already exists add one
+    if (!appWrapper.size()) {
+      // AppWrapper tags
+      const appWrapperOpeningTag = j.jsxOpeningElement(j.jsxIdentifier(appWrapperIdentifier));
+      const appWrapperClosingTag = j.jsxClosingElement(j.jsxIdentifier(appWrapperIdentifier));
+
+      // wrap children in AppWrapper tags
+      dialogFullScreen.value.children = [
+        appWrapperOpeningTag,
+        ...dialogFullScreen.value.children,
+        appWrapperClosingTag
+      ];
+
+      return true;
+    } else {
```

Comment:
I don't think this else block is needed either, the function returning undefined will be enough looking at the logic below